src/AppInstallerCLICore/Workflows/ConfigurationFlow.cpp

```diff
@@ -1536,15 +1574,27 @@ namespace AppInstaller::CLI::Workflow
         {
             ConfigurationContext& configContext = context.Get<Data::ConfigurationContext>();
 
+            // PowerShell package needs to be present for certain predefined modules to work.
+            ConfigurationUnit powerShellPackageUnit = CreatePowerShellPackageUnit();
+            configContext.Set().Units().Append(powerShellPackageUnit);
+
+            // Apply the unit to make sure it's on the system.
+            context.Reporter.Info() << Resource::String::ConfigurationExportInstallRequiredModule(Utility::LocIndView{ "Microsoft PowerShell Package" }) << std::endl;
+            auto applyPowerShellResult = ApplyUnit(context, powerShellPackageUnit);

```

Comment:
We always apply it without testing?

Reply:
For the wingetpackage dsc resource, it tests it (in the resource implementation) before applying. So I'm just applying it to save 2 calls.